### PR TITLE
Adicionado link do CV

### PR DESCRIPTION
**Pull request by alfredudu**
Hello Edy, I added link to my CV

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <body>
     <h1>All Students</h1>
     <ul>
-        <li>Alfredo Eduardo da Silva</li>
+        <li><a href="https://www.linkedin.com/in/alfredoesilva"</li>Alfredo Eduardo da Silva</a>
```
Comment (JessicaAdriele) [inline index.html]: Please fix the closing tags

Comment (edysegura) [reply index.html]: Good catch! Please @alfredudu fix that!

**Comment by JessicaAdriele** [comment]
:+1: thanks

File: index.html (hunk 1 above)
Comment (alfredudu) [reply index.html]: Fixed with https://github.com/univas/all-students/pull/3/commits/fa7588d269445035da41fee0dce729f697e00053

**Comment by edysegura** [comment]
Awesome!!!

**Pull request merged**